.travis.yml

```diff
@@ -1,21 +1,38 @@
+import:
+  - twingly/travis-ci:config/wait-for-mysql.yml@master
+
 dist: bionic
 language: ruby
 services:
   - docker
+
+matrix:
+  include:
+    - env:
+      - MYSQL_VERSION="5-6"
+    - env:
+      - MYSQL_VERSION="5-7"
+    - env:
+      - MYSQL_VERSION="8"
```

Comment:
I think the following is enough:

```yml
- env:
  - MYSQL_VERSION="5-6"
- env:
  - MYSQL_VERSION="5-7"
- env:
  - MYSQL_VERSION="8"
```

`env` gives you an "implicit" matrix

Reply:
Oh, you mean to remove this part?
```
matrix:
  include:
```

Reply:
yea!

Reply:
and sorry, I think it should be like this:

```yml
env:
  - MYSQL_VERSION="5-6"
  - MYSQL_VERSION="5-7"
  - MYSQL_VERSION="8-0"
```